src/Functions/FunctionsTimeWindow.cpp

```diff
@@ -684,7 +684,7 @@ ColumnPtr FunctionTimeWindow<type>::executeImpl(const ColumnsWithTypeAndName & a
     return TimeWindowImpl<type>::dispatchForColumns(arguments, name);
 }
 
-void registerFunctionsTimeWindow(FunctionFactory& factory)
+__attribute__((used)) void registerFunctionsTimeWindow(FunctionFactory& factory)
```

Comment:
That should be FUNCTION_REGISTER, right?

Reply:
Completed